config/libc/musl.in

```diff
@@ -1,6 +1,7 @@
 # musl options
 
 ## depends on ! WINDOWS && ! BARE_METAL
+## depends on EXPERIMENTAL
```

Comment:
An alternate solution would have been to have musl depend on the
archictectures we know it works on (or excluded on the architectures
we know are broken).

Right now, Buildroot considered musl to be no longer experimental,
so it's weird we would do the opposite... But indeed, Buildroot has
a rather strict set of architectures on which musl is supposed to be
supported:
    https://git.buildroot.org/buildroot/tree/toolchain/toolchain-buildroot/Config.in#n89

But as you wrote, when/if someone is that interested in musl, he can
come up with a patch to fix the situation (I'm not much interested
myself...).

Reviewed-by: "Yann E. MORIN" <yann.morin.1998@free.fr>